src/javascript/detectors/insecure_cookie/insecure_cookie.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=insecure-cookie@v1.0 defects=1}
+var cookieSession = require('cookie-session')
+var express = require('express')
+var app = express()
+function insecureCookieNoncompliant() {
+    let session =
+    app.use(cookieSession({
```

Comment:
Formatting is a bit strange here, can we remove the new line?

Reply:
removed new line